core-api/src/main/java/org/codice/imaging/nitf/core/RGBColour.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ */
+package org.codice.imaging.nitf.core;
+
+import java.awt.Color;
+
+/**
+ Red / Green / Blue colour representation.
+ */
+public interface RGBColour {
+    /**
+     Return the red component of the colour.
+
+     @return red component of the colour
+     */
+    byte getRed();
```

Comment:
If its an easy fix (i.e. the IDE will redo it for you), can you  change these to standard javadoc?
So
```
    /**
     * Return the red component of the colour.
     *
     * @return red component of the colour
     */
```
but everywhere.

If you have to do it by hand, then its up to you whether you have anything else to do...